api/specs/web-server/_groups.py

```diff
@@ -113,6 +119,7 @@ async def add_group_user(
 @router.get(
     "/groups/{gid}/users/{uid}",
     response_model=Envelope[GroupUserGet],
+    tags=_extra_tags,
```

Comment:
This was already there. Probably from your initial version. A use case for this might be a particular user in an organization.

Reply:
no I pretty much doubt it. but whatever, it is lost in the refactoring.

Reply:
I can trace it back since a very old version :-)

https://github.com/ITISFoundation/osparc-simcore/blob/v1.34.1/services/web/server/src/simcore_service_webserver/groups_handlers.py#L141-L154